docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+| R6   | Unable to control payments                                 | critical | <ul><li>Ability to control when, who & how much to pay</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+<!-- prettier-ignore-end -->
+
+#### Basic Test Cases
+
+##### Connect to a wallet
+
+**Test ID**: 1  
+**Description**: Connect the extension to a Web Monetization enabled digital wallet
+**Risk**: R2 (onboarding) and R5 (UX)  
+**Preconditions**:
+
+1. You have already downloaded and installed the Web Monetization extension.
+2. Your digital wallet sign-up and account activation is complete (learn more from the [Prerequisites](#Prerequisites) section)
+
+**Steps**:
+
+1. Open the WM extension and copy the extension’s key.
+2. Open the digital wallet, and load the extension’s key into the wallet.
+3. From the digital wallet, find and copy the wallet address or payment pointer.
+4. Open the WM extension, and enter the wallet address or payment pointer.
+5. Enter the amount you want to make available to the extension from your wallet.
+6. Keep the option to renew monthly disabled.
+
+**Expected results**:
+
+1. The wallet owner receives an interaction prompt from their wallet to authorize the connection and access to the amount.
+2. On accepting, you get shown the message "Your wallet is now successfully connected to the extension."
+
+##### Make continuous payments
+
+**Test ID**: 2  
+**Description**: Send continuous WM payments to a monetized website  
+**Risk**: R3 (security of funds)  
+**Preconditions**:
+
+1. The extension is connected to your wallet.
+2. The extension has a positive remaining balance.
+3. Your wallet balance is equal to or greater than the extension’s remaining balance.
+
+**Steps**:
+
+1. Visit a monetized website. Refer to the [Test Data](#Test-Data) section to explore different WM conditions for websites.
+2. Visit a non-monetized website. Open the extension to observe its available options.
```

Comment:
We should suggest a URL or site. Give them a little more direction. We could say "Visit a non-web monetized website, such as your favorite search engine."